uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-assignment.controller.spec.js

```diff
@@ -272,6 +282,147 @@ define([
       });
     });
 
+    describe('when the target contact changes', function () {
+      var assigneeId = 5;
+
+      beforeEach(function () {
+        scope.assignment.client_id = 1;
+        scope.assignment.contact_id = 2;
+        scope.taskList = [ { id: 3 } ];
+        scope.activity.activitySet = {
+          activityTypes: [{
+            id: 4,
+            default_assignee_type: defaultAssigneeOptionsIndex.SPECIFIC_CONTACT.value,
+            default_assignee_contact: assigneeId
+          }]
+        };
+
+        scope.onTargetContactChange();
+        $rootScope.$digest();
+      });
+
+      it('assigns the contact id to the client id', function () {
+        expect(scope.assignment.client_id).toBe(scope.assignment.contact_id);
+      });
+
+      it('loads the default assignees related to the new target contact', function () {
+        expect(scope.taskList[0].assignee_contact_id).toEqual([assigneeId]);
+      });
+    });
+
+    describe('selecting the default assignee', function () {
+      var activityType;
+      var loggedInContactId = 7891011;
```

Comment:
Instead of all these arbitrary values, can we use `_.uniqueId()`? (please do this in all places)